geo/tests/jts_tests.rs

```diff
@@ -13,7 +13,7 @@ fn init_logging() {
 fn test_all_general() {
     init_logging();
 
-    let mut runner = TestRunner::new();
+    let mut runner = TestRunner::new().with_precision_floating();
```

Comment:
Currently the precision check is done per file (as the precision is available at the top of the xml file).  This may end-up not running some tests that would any way pass.  eg. Relate tests seem to have a non-default precision model, but being a boolean predicate, I think the model doesn't matter?